modules/core/02-client/keeper/events.go

```diff
@@ -82,13 +82,12 @@ func EmitUpgradeClientEvent(ctx sdk.Context, clientID string, clientState export
 }
 
 // EmitUpdateClientProposalEvent emits an update client proposal event
-func EmitUpdateClientProposalEvent(ctx sdk.Context, clientID string, clientState exported.ClientState) {
+func EmitUpdateClientProposalEvent(ctx sdk.Context, clientID, clientType string) {
 	ctx.EventManager().EmitEvent(
 		sdk.NewEvent(
 			types.EventTypeUpdateClientProposal,
 			sdk.NewAttribute(types.AttributeKeySubjectClientID, clientID),
-			sdk.NewAttribute(types.AttributeKeyClientType, clientState.ClientType()),
-			sdk.NewAttribute(types.AttributeKeyConsensusHeight, clientState.GetLatestHeight().String()),
```

Comment:
There's no way to guarantee the latest height of the substitute is the latest height of subject. I question the usefulness of this information, so I'd prefer to remove it for now until it is requested (at which time we can query the subject client state)